DIPs/DIP1xxx-JS.md

```diff
@@ -66,21 +83,27 @@ code if refactoring is not possible.
 
 ### Language Features
 
-Deprecations to language features must also update the [language deprecations
-page](https://dlang.org/deprecate.html) on dlang.org simultaneously.
+Unless the removed language feature is very unsafe or causes damage to real
+world systems, all changes or removals must be accompanied with a deprecation
+period. "Language features" includes bugs in the current behavior that existing
+user code depends on, e.g. [Issue 10378](https://issues.dlang.org/show_bug.cgi?id=10378).
+Fixing such issues should include a deprecation preiod for the current behavior,
```

Comment:
Typo: Period

Reply:
Do you mean this sentence should be broken up?

Reply:
No, you misspelled period. You wrote preiod.

Reply:
Oh, ha. Totally missed that.

Reply:
I did too until Seb pointed it out.